docs/software/soroban-settings.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Soroban Settings
+---
+
+Soroban has a large set of settings stored on ledger than can be modified
+through a vlaidator vote. This documents describes how to propose a new settings
```

Comment:
typo: validator
nit: This document describes

Reply:
Fixed